airmozilla/manage/static/manage/js/survey-edit.js

```diff
@@ -31,19 +31,28 @@ $(function() {
     // hijack the ability to save/edit
     $('form.edit').submit(function() {
         var form = $(this);
+        $(form.children('button')).text('Saving..');
```

Comment:
Can you make that an ellipsis instead. `…`
It's a [Unicode character](https://en.wikipedia.org/wiki/Ellipsis) and it might not be clear how to copy that. But if I paste in what I can put in my editor it looks like this:
```javascript
$(form.children('button')).text('Saving…');
```